core/src/main/java/org/elasticsearch/index/query/MatchQueryBuilder.java

```diff
@@ -153,6 +160,9 @@ protected void doWriteTo(StreamOutput out) throws IOException {
         out.writeOptionalString(fuzzyRewrite);
         out.writeOptionalWriteable(fuzziness);
         out.writeOptionalFloat(cutoffFrequency);
+        if (out.getVersion().onOrAfter(Version.V_7_0_0_alpha1)) {
```

Comment:
Right, it is too late to add this to 5.x so the target for this new feature is 6.1 and 7.